src/main/java/com/cloudbees/hudson/plugins/folder/computed/ComputedFolder.java

```diff
@@ -108,6 +108,12 @@ protected ComputedFolder(ItemGroup parent, String name) {
         init();
     }
 
+    protected ComputedFolder(ItemGroup parent, String name, OrphanedItemStrategy strategy) {
```

Comment:
:ant: Works for this one purpose, but more general and flexible to revert the new constructor and instead add `setOrphanedItemStrategy`.